Japanese blog two year celebration english including all contributor's review (#4240)
OpenLiberty/blogs: posts/2024-12-16-jblog-two-year-celebration.adoc

---
layout: post

> title: The OpenLiberty Japanese blog has celebrated its second anniversary

Reviewer: ```suggestion
title: The Open Liberty Japanese blog has celebrated its second anniversary
```

# Do NOT change the categories section
categories: blog
author_picture: https://avatars3.githubusercontent.com/una-tapa
author_github: https://github.com/una-tapa

> seo-title: The OpenLiberty Japanese blog has celebrated its second anniversary

Reviewer: ```suggestion
seo-title: The Open Liberty Japanese blog has celebrated its second anniversary
```

> seo-description: The OpenLiberty Japanese blog has reached its second anniversary since its launch in 2022. We will introduce the most popular articles to date and highlight the contributors who have supported us.

Reviewer: ```suggestion
seo-description: The Open Liberty Japanese blog has reached its second anniversary since its launch in 2022. We will introduce the most popular articles to date and highlight the contributors who have supported us.
```

> blog_description: OThe OpenLiberty Japanese blog has reached its second anniversary since its launch in 2022. We will introduce the most popular articles to date and highlight the contributors who have supported us

Reviewer: ```suggestion
blog_description: The Open Liberty Japanese blog has reached its second anniversary since its launch in 2022. We will introduce the most popular articles to date and highlight the contributors who have supported us
```

open-graph-image: https://openliberty.io/img/twitter_card.jpg
open-graph-image-alt: Open Liberty Logo
blog-available-in-languages:
- lang: ja
  path: /ja/blog/2024/12/16/jblog-two-year-celebration.html
additional_authors:
- name: Takakiyo Tanaka
  github: https://github.com/takakiyo
  image: https://avatars.githubusercontent.com/takakiyo
- name: Tsuyoshi Baba
  github: https://github.com/babatch
  image: https://avatars.githubusercontent.com/babatch
- name: Kaori Asada
  github: https://github.com/kaori-asa
  image: https://avatars.githubusercontent.com/kaori-asa
- name: Atsushi Kurokawa
  github: https://github.com/akurokawa
  image: https://avatars.githubusercontent.com/akurokawa
- name: Miyuka Nishio
  github: https://github.com/MiyukaNishio
  image: https://avatars.githubusercontent.com/MiyukaNishio
---

> = The OpenLiberty Japanese blog has celebrated its second anniversary

Reviewer: ```suggestion
= The Open Liberty Japanese blog has celebrated its second anniversary
```

Hiroko Takamiya <https://github.com/una-tapa>

:imagesdir: /
:url-prefix:
:url-about: /
//Blank line here is necessary before starting the body of the post.

> Launched in 2022, the OpenLiberty Japanese blog was created to meet the increasing demand for localized OpenLiberty content in Japan. 

Reviewer: ```suggestion
Launched in 2022, the Open Liberty Japanese blog was created to meet the increasing demand for localized Open Liberty content in Japan. 
```

Hello, I’m Miyuka Nishio, and I’m currently in my second year at IBM. I serve as a technical sales specialist for WebSphere Liberty and Open Liberty, focusing on pre-sales activities and community management. My background was primarily in accounting before I joined IBM, so I had little experience with Java development. However, my manager encouraged me to get involved in translating the Open Liberty blog as part of my professional development. This opportunity has enabled me to learn about Liberty's new features and strengthen my engineering skills. There’s still much for me to learn, but I am eager to continue growing as an engineer and contributing meaningfully to the Open Liberty project.

link:https://www.linkedin.com/in/tsuyoshi-baba-babatch[Tsuyoshi Baba]

I joined IBM Japan in 1998 and began working with WebSphere soon after the release of WebSphere Application Server V1.1 in Japan that same year. Initially, I was involved in product marketing before transitioning to technical sales, where I supported our business partners. In 2022, I returned to IBM Japan and now focus on Application Modernization, including WebSphere, as a technical sales specialist.

> I find the top article rankings very intriguing. It seems timeless that once someone gets an application server running, the next step is always to connect it to a database and start building applications. Announcements about new versions of Java/Jakarta EE, Java SE, and MicroProfile are also popular. And of course, Spring continues to be a strong presence! The popularity of articles about ARM64 containers might be due to the increasing number of developers using the new Apple Silicon-equipped Macs—I'm one of them too.

Reviewer: ```suggestion
I find the top article rankings very intriguing. It seems timeless that once someone gets an application server running, the next step is always to connect it to a database and start building applications. Announcements about new versions of Java EE/Jakarta EE, Java SE, and MicroProfile are also popular. And of course, Spring continues to be a strong presence! The popularity of articles about ARM64 containers might be due to the increasing number of developers using the new Apple Silicon-equipped Macs—I'm one of them too.
```

[#backend]
== Meet the Team Behind the Blog

link:https://openliberty.io/blog/2021/12/10/DavidMueller_MeetTheTeam.html[David Mueller]

David is the leader of the Liberty documentation team. He always makes him available to consult on anything related to the Liberty blog articles.

link:https://github.com/kinueng[Kin Ueng], link:https://github.com/steven1046[Steven Zvonek], link:https://github.com/OpenLiberty/blogs/pulls/natalie-bernhard[Natalie Bernhard] :

Kin, Steven, and Natalie are the key figures who have set up the blog posting system and built multilingual cross-links. They have also created Japanese banners and buttons!

link:https://github.com/yeekangc[Yee-Kang Chang], link:https://github.com/GraceJansen[Grace Jansen], link:https://github.com/Emily-Jiang[Emily Jiang]

> YK, as an Architect in Developer Advocacy, and Emily and Grace, as Java Champions, are supporting the Japanese OpenLiberty blog as part of their worldwide community activities.

Reviewer: ```suggestion
YK, as an Architect in Developer Advocacy, and Emily and Grace, as Java Champions, are supporting the Japanese Open Liberty blog as part of their worldwide community activities.
```

[source,xml]
----
<dependency>
    <groupId>org.example.spec</groupId>
    <artifactId>exampleApi</artifactId>
    <version>7.0</version>
    <type>pom</type>
    <scope>provided</scope>
</dependency>
<dependency>
    <groupId>example.platform</groupId>
    <artifactId>example.example-api</artifactId>
    <version>11.0.0</version>
    <scope>provided</scope>
</dependency>
----

Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:

[source,gradle]
----
buildscript {
    repositories {
        mavenCentral()
    }
    dependencies {
        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.9.1'
    }
}
apply plugin: 'liberty'
dependencies {
    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[25.0.0.1-beta,)'
}
----

Or if you're using link:{url-prefix}/docs/latest/container-images.html[container images]:

[source]
----
FROM icr.io/appcafe/open-liberty:beta
----

Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].

> If you're using link:https://plugins.jetbrains.com/plugin/14856-liberty-tools[IntelliJ IDEA], link:https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Visual Studio Code] or link:https://marketplace.eclipse.org/content/liberty-tools[Eclipse IDE], you can also take advantage of our open source link:{url-prefix}/docs/latest/develop-liberty-tools.html[Liberty developer tools] to enable effective development, testing, debugging, and application management all from within your IDE.

Reviewer: ```suggestion
If you're using link:https://plugins.jetbrains.com/plugin/14856-liberty-tools[IntelliJ IDEA], link:https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Visual Studio Code] or link:https://marketplace.eclipse.org/content/liberty-tools[Eclipse IDE], you can also take advantage of our open source link:{url-prefix}/docs/latest/develop-liberty-tools.html[Liberty Tools] to enable effective development, testing, debugging, and application management all from within your IDE.
```
